src/test/java/uk/ac/ebi/embl/api/validation/helper/ReferenceUtilsTest.java

```diff
@@ -0,0 +1,122 @@
+package uk.ac.ebi.embl.api.validation.helper;
+
+
+import org.easymock.EasyMock;
+import org.junit.Test;
+import uk.ac.ebi.embl.api.entry.reference.Person;
+import uk.ac.ebi.embl.api.entry.reference.Reference;
+import uk.ac.ebi.embl.api.entry.reference.Submission;
+import uk.ac.ebi.embl.api.validation.ValidationEngineException;
+import uk.ac.ebi.embl.api.validation.dao.entity.ReferenceEntity;
+import uk.ac.ebi.embl.flatfile.writer.FlatFileWriter;
+
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.replay;
+import static org.junit.Assert.*;
+
+public class ReferenceUtilsTest  {
+
+    ReferenceUtils refutils = new ReferenceUtils();
+
+    @Test
+    public void testConstructReferenceWithoutConsortiumAndBroker() throws UnsupportedEncodingException {
+
+        List<ReferenceEntity> refEntityList = new ArrayList<>();
+        ReferenceEntity refEntity = new ReferenceEntity();
+        refEntity.setBrokerName(null);
+        refEntity.setSubmissionAccountId("Webin-256");
+        refEntity.setConsortium(null);
+        refEntity.setSurname("Vijay");
+        refEntity.setMiddleInitials(null);
+        refEntity.setFirstName("Senthil");
+        refEntity.setCenterName("EBI.(EBL-EBI)");
+        refEntity.setFirstCreated("22-SEP-2020");
+        refEntity.setLaboratoryName("EBI -EMBL");
+        refEntity.setAddress("Hinxton, Cambridgeshire");
+        refEntity.setCountry("United Kingdom");
+        refEntityList.add(refEntity);
+
+        Reference reference = refutils.constructReference(refEntityList);
+        assertNotNull(reference);
+        assertEquals("EBI -EMBL", ((Submission) reference.getPublication()).getSubmitterAddress());//just labarotary name for non-brokers
```

Comment:
This test should fail because it only has laboratory name. Should also have center name etc.